bootstrap/_rpm_common.sh

```diff
@@ -21,7 +21,6 @@ $tool install -y \
   python \
   python-devel \
   python-virtualenv \
-  python-devel \
```

Comment:
@devnsec-com: since we might not merge this PR at all, can you submit another with this change only?